server/middleware/populateCurrentUser.test.ts

```diff
@@ -44,10 +45,25 @@ describe('populateCurrentUser', () => {
     expect(next).toHaveBeenCalled()
   })
 
+  it('should populate the current user from the API if the user version not equals', async () => {
+    ;(userService.getActingUser as jest.Mock).mockResolvedValue(user)
+    request.session.user = user
+    const middleware = populateCurrentUser(userService)
+
+    await middleware(request, response, next)
+
+    expect(request.session.user).toEqual(user)
+    expect(response.locals.user).toEqual({ ...response.locals.user, ...user })
+
+    expect(userService.getActingUser).toHaveBeenCalledWith(token)
+    expect(next).toHaveBeenCalled()
+  })
+
   it('should fetch the user from the session if present', async () => {
```

Comment:
might `should fetch the user from the session if present and the user version matched the value in the inMemoryStore` be more accurate?